examples/kitchen_sink_proc_macro.rs

```diff
@@ -198,7 +204,9 @@ pub fn main() {
     {
         let cache = CUSTOM.lock().unwrap();
         println!("hits: {:?}", cache.cache_hits());
+        assert_eq!(cache.cache_hits().unwrap(), 25);
         println!("misses: {:?}", cache.cache_misses());
+        assert_eq!(cache.cache_misses(), Some(1));
```

Comment:
Could you change this to assert None for both of these and also add a comment saying something along the lines of "custom cache doesn't implement these so they're none"?